Assets/_System/Scripts/TurnBasedScripts/CombatPlayer.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+[Serializable]
+public class CombatPlayer
+{
+    public List<GridObject> units;
```

Comment:
Do these things need to be public for other things to modify?

Reply:
Yes we need to index into it as well as modify the specific units